docker-compose.dev.yaml

```diff
@@ -0,0 +1,37 @@
+version: "3.7"
+services:
+  #############################################################################################
+  ###                                         API                                           ###
+  #############################################################################################
+  dpia-api:
+    container_name: dpia-api
+    build:
+      context: .
+      dockerfile: ./src/backend/.docker/Dockerfile.dev
```

Comment:
Would you be creating one for dev and one for production? Preferably have one if possible and have it in the src/backend/Dockerfile path?

Reply:
The idea is `Dockerfile.dev` is actually to be used just for local development - for hot reload etc. And the other Dockerfile should be used for dev, test and prod deployments.

But looks like the naming convention could be a bit misleading - I can make it Dockerfile.local if that makes sense?